net/aircrack-ng/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2015 OpenWrt.org
+# Copyright (C) 2006-2018 OpenWrt.org
```

Comment:
You can't assign copyright unless you're affiliated
Please revert this change and add your own (if you want)